rest/utilities_testing.go

```diff
@@ -674,6 +674,9 @@ type BlipTesterSpec struct {
 	// Run Sync Gateway in "No conflicts" mode.  Will be propgated to the underyling RestTester
 	noConflictsMode bool
 
+	// If an underlying RestTester is created, it will propagate this setting to the underlying RestTester.
+	guestEnabled bool
```

Comment:
This doesn't look like it's actually wired up to anything on this branch - can it just be omitted from the test?

Reply:
So this was added as `TestProposedChangesIncludeConflictingRev` uses guestEnabled in the blip tester spec on this branch

Reply:
It sets it but nothing reads this value
Does that mean there is missing functionality inside BlipTester that needs to be pulled in as well?
Or is the test setting something it doesn't need to?

Reply:
Ah I see good point. I've just removed it and doesn't effect the test when I've run it against walrus and server so will push those changes up